modules/ROOT/partials/configuration/images_upload_handler.adoc

```diff
@@ -25,6 +25,7 @@ include::partial$misc/images-upload-handler-function.adoc[]
 
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
+  plugins: 'image',
```

Comment:
This is a core option and not tied to the image plugin so we shouldn't be listing this here.

- [ ] Revert this as core options shouldn't have plugins listed. If it's needed for certain pages then it this line should be conditional.

Reply:
Fixed